src/System.Security.Cryptography.Algorithms/tests/DSATests.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using Xunit;
+
+namespace System.Security.Cryptography.Algorithms.Tests
+{
+    public class DSATests
+    {
+        [Fact]
+        public void BaseVirtualsNotImplementedException()
+        {
+            var dsa = new EmptyDSA();
+            Assert.Throws<NotImplementedException>(() => dsa.HashData(null, HashAlgorithmName.SHA256));
+            Assert.Throws<NotImplementedException>(() => dsa.HashData(null, 0, 0, HashAlgorithmName.SHA256));
```

Comment:
Are there tests for the new default span methods which defer to the existing array implementations?  (They might currently have code coverage from DSACryptoServiceProvider, but I'd like to not need to count on that.)